code/modules/research/designs/weapons.dm

```diff
@@ -98,9 +98,9 @@
 	name = "Assault Rifle"
 	desc = "An Assault Rifle."
 	id = "xcomar"
-	req_tech = list("combat" = 4, "materials" = 3)
+	req_tech = list("combat" = 5, "materials" = 4)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 10000, MAT_GLASS = 1000)
+	materials = list(MAT_IRON = 10000, MAT_GLASS = 1000, MAT_SILVER = 4000, MAT_DIAMOND = 2000)
```

Comment:
I do not think it's overkill, automatic weapons are very strong. I've used the AR myself in the past for traitoring because of how cheap and easy to get it is. Diamonds would be a minimum for me.